library/src/main/java/com/novoda/downloadmanager/lib/DownloadReadyChecker.java

```diff
@@ -37,6 +37,7 @@ private boolean isDownloadManagerReadyToDownload(FileDownloadInfo downloadInfo)
         }
         switch (downloadInfo.getStatus()) {
             case 0: // status hasn't been initialized yet, this is a new download
+            case DownloadStatus.SUCCESS:
```

Comment:
when a  download is completed, we actually attempt to restart it again anyway - because we request *all* downloads in a batch not just uncomplete ones. This is a known issue - we came across it in pause/resume